app/renderer/components/Inspector/SelectedElement.js

```diff
@@ -141,6 +143,48 @@ class SelectedElement extends Component {
       showXpathWarning = true;
     }
 
+    // Add class chain to the data source as well
+    if (classChain && currentContext === NATIVE_APP) {
+      const classChainText = <Tooltip title={t('This selector is in BETA, it is the XML selector translated to `-ios class chain`.')}>
+        {/* eslint-disable-next-line shopify/jsx-no-hardcoded-content */}
+        <span>
+          -ios class chain
+          <strong>
+            {/* eslint-disable-next-line shopify/jsx-no-hardcoded-content */}
+            <a onClick={(e) => e.preventDefault() || shell.openExternal('https://github.com/facebookarchive/WebDriverAgent/wiki/Class-Chain-Queries-Construction-Rules')}>(beta)</a>
+          </strong>
+        </span>
+      </Tooltip>;
+
+      findDataSource.push({
+        key: '-ios class chain',
+        find: classChainText,
+        selector: classChain,
+        time: getTimeButton
+      });
+    }
+
+    // Add predicate string to the data source as well
+    if (predicateString && currentContext === NATIVE_APP) {
+      const predicateStringText = <Tooltip title={t('This selector is in BETA, it is the XML selector translated to `-ios predicate string`.')}>
```

Comment:
is this tooltip still relevant?

Reply:
I think so, it's still a beta, but happy to remove it or change the description if needed. Same is for the `class chain` selector, that one has the same type of description